modules/loader-utils/src/lib/library-utils/library-utils.js

```diff
@@ -36,12 +36,19 @@ export function getLibraryUrl(library, moduleName, options) {
     return modules[library];
   }
 
+  // Load from local files, not from CDN scripts in Node.js
+  // TODO - needs to locate the modules directory when installed!
+  if (!isBrowser) {
+    return `modules/${moduleName}/dist/libs/${library}`;
```

Comment:
Since this is inside util module which will be shared by other modules, better to have stable way to resolve the module directory. Add an issue to track this?

Reply:
Yes good point, the dynamic module loading was a bit rushed in order to get ready for v2.0. 

I will put together a separate, smaller PR with test cases for the library-util functions and we can iterate on design etc there.